json-rpc/src/counters.rs

```diff
@@ -48,6 +49,7 @@ pub static INVALID_REQUESTS: Lazy<IntCounterVec> = Lazy::new(|| {
             "method", // method of request, matches JSON RPC method name (e.g. "submit", "get_account")
             "errortype", // categories of invalid requests: "invalid_format", "invalid_params", "invalid_method", "method_not_found"
             "sdk_lang",  // the language of the SDK: "java", "python", etc
+            "sdk_vers",  // the version of the SDK: "0.0.11", "1.45.31", etc
```

Comment:
small nit: should we have this be sdk_ver since it refers to a specific version instead of multiple versions?